megamek/src/megamek/client/ui/clientGUI/boardview/BoardView.java

```diff
@@ -1748,6 +1762,32 @@ private Mounted<?> selectedWeapon() {
         return (clientgui != null) ? clientgui.getDisplayedWeapon().orElse(null) : null;
     }
 
+    /**
+     * Draws white hex borders for highlighted entity hexes (Nova CEWS network dialog).
+     *
+     * @param graphics The graphics object to draw on
+     */
+    private void drawEntityHexHighlights(Graphics2D graphics) {
+        if (highlightedEntityHexes.isEmpty()) {
+            return;
+        }
+
```

Comment:
This block is unnecessary as the for loop while not do anything anyway if the list of hexes is empty and it is not worth returning before the setColor call.

Reply:
Addressed as part of https://github.com/MegaMek/megamek/pull/7635/commits/3ff983a99dc7e832f0d045cbe5ddc06d9aa89f3c